win720.py

```
@@ -52,69 +54,110 @@ def buy_Win720(
         auth_ctrl: auth.AuthController,
         username: str
     ) -> dict:
-        assert type(auth_ctrl) == auth.AuthController
+        assert isinstance(auth_ctrl, auth.AuthController)
 
         headers = self._generate_req_headers(auth_ctrl)
```

Ruff (0.14.10)

Local variable `headers` is assigned to but never used

Remove assignment to unused variable `headers`

(F841)